lightning-invoice/src/lib.rs

```diff
@@ -795,18 +795,18 @@ impl SignedRawInvoice {
 /// variant. If no element was found `None` gets returned.
 ///
 /// The following example would extract the first B.
-/// ```
-/// use Enum::*
+/// ``
+/// use Enum::*;
```

Comment:
May as well have kept the fully-qualified enum variants as I don't think this was ever valid rust. 😛